src/main/java/org/myrobotlab/service/abstracts/AbstractSpeechRecognizer.java

```diff
@@ -570,18 +567,25 @@ public void unsetWakeWord() {
   }
 
   @Override
-  public void setLocale(String code) {
-    locale = new Locale(code);
+  public String getLanguage() {

```

Comment:
The intention of this method is to tell you what language the speech recognizer is detecting.  it's is not the same as Runtime locale.   This seems wrong to me.

Reply:
no .. its original intention was to support the Locale.  And its interface supported changing and getting Locale information including language

Reply:
This should be the language that the speech recognizer is recognizing in.  I don't think this is expected behavior.

Reply:
updated - pushed to Service - each service has its own Locale